code/common/eodtime.q

```diff
@@ -32,3 +32,12 @@ getday:{[p]
 
 d:getday[.z.p];                                             // get current date when loading process, store in d
 nextroll:getroll[.z.p];                                     // get next roll when loading process, store in nextroll
+
+currperiod:0D01 xbar .z.p                                   // current logging period (for stp process)
```

Comment:
0D01 should be a variable (we might want to do 15 minute log files, for example)